### PR TITLE
[GH-134] Add lhs

### PR DESCRIPTION
**Pull request by iomodo**
This PR adds left hand side support. Code in LHSNavigation is just for testing and will be removed in later PRs

https://github.com/oseducation/knowledge-graph/assets/2340127/230a1568-82c3-4389-95df-57dde8cf8a50

Fixes: #134

### CODE REVIEW

File: webapp/src/components/lhs/item.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
+        setAnchorEl(null);
+    };
+
+    if (props.isCollapsed) {
```
Comment (gbochora) [inline webapp/src/components/lhs/item.tsx:28]: Why should the item know if the group is collapsed or not?

File: webapp/src/components/lhs/group.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            >
+                <Typography
+                    width={'100%'}
+                    fontSize={14}
+                    fontWeight={600}
+                >
+                    {props.group.display_name}
+                </Typography>
+            </HeaderButton>
+            <List dense={true}>
```
Comment (gbochora) [inline webapp/src/components/lhs/group.tsx:46]: Would it be better to include the `List` component only when the group is not collapsed?

File: webapp/src/components/lhs/item.tsx (hunk 1 above)
Comment (iomodo) [reply webapp/src/components/lhs/item.tsx:28]: If group is collapsed all items are collapsed too

**Review by gbochora — APPROVED** [review]

File: webapp/src/components/lhs/group.tsx (hunk 2 above)
Comment (iomodo) [reply webapp/src/components/lhs/group.tsx:46]: good point

**Pull request merged**